app/scripts/controllers/visualizerModal.js

```diff
@@ -12,22 +12,16 @@ angular.module('openhimConsoleApp')
     $scope.validationRequiredMsg = 'This field is required.';
 
     // get/set the users scope whether new or update
-    if (settingsStore) {
-      $scope.update = true;
-      
-      
-    }else{
-      $scope.update = false;
+    $scope.update = false;
 
-      // create visualizer settings properties
-      $scope.settings = {};
-      $scope.settings.visualizer = {};
+    // create visualizer settings properties
+    $scope.settings = {};
+    $scope.settings.visualizer = {};
```

Comment:
Maybe I'm just being picky, but can't we rather store the visualizer setting in `$scope.visualizer` there is no need for the extra `.settings` object. It just makes the code look a bit cleaner.